assets/models/system/automl-ner/description.md

```diff
@@ -0,0 +1,89 @@
+* Either download or deploy the resulting model as a endpoint in Azure Machine Learning.
+* Scale the operationalization process with the help of Azure Machine Learning's MLOps and ML Pipelines capabilities.
+
+See [How to train nlp models](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=cli) for more information.
+
+### Documentation
+
+#### Prepare Data
+To create NLP models, it is necessary to provide labeled text data as input for model training. Named entity recognition requires CoNLL format. The file must contain exactly two columns and in each row, the token and the label is separated by a single space. Example as below:
+
+```txt
+Hudson B-loc
+Square I-loc
+is O
+a O
+famous O
+place O
+in O
+New B-loc
+York I-loc
+City I-loc
+
+Stephen B-per
+Curry I-per
+got O
+three O
+championship O
+rings O
+```
+
+Please see [documentation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=python#named-entity-recognition-ner) for data requirements.
+
+#### Language Setting
+
+Currently, language selection defaults to English. But Automated ML supports 104 languages leveraging language specific and multilingual pre-trained text DNN models. Please see [langugage setting](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=python#language-settings) for documentation.
+
+#### Train a Model
+
+You can initiate [individual trials](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=cli#data-validation), or perform a [manual sweeps](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=python#model-sweeping-and-hyperparameter-tuning-preview), which explores multiple hyperparameter values near the more promising models and hyperparameter configurations. 
+
+For more information, see [Model sweeping and hyperparameter tuning](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=python#model-sweeping-and-hyperparameter-tuning-preview).
+
+### Code samples
+
+
+Task|Dataset|Python sample (Notebook)|CLI with YAML
+|---|--|--|--|
+Named Entity Recognition|[CoNLL-2003](https://www.kaggle.com/datasets/alaakhaled/conll003-englishversion?select=valid.txt)|<a href="https://github.com/Azure/azureml-examples/blob/main/sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-named-entity-recognition-task-distributed-sweeping/automl-nlp-text-ner-task-distributed-with-sweeping.ipynb" target="_blank">automl-nlp-text-ner-task-distributed-with-sweeping.ipynb</a>|<a href="https://github.com/Azure/azureml-examples/blob/main/cli/jobs/automl-standalone-jobs/cli-automl-text-ner-conll-distributed-sweeping/cli-automl-text-ner-conll2003-distributed-sweeping.yml" target="_blank">/automl-nlp-text-ner-task-distributed-with-sweeping.ipynb</a>
```

Comment:
the link text for cli example is sdk not cli

Reply:
thank you for catching that! you have good eyes!